.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,146 @@
+name: "Continuous Integration"
+
+on: [push, pull_request]
+
+jobs:
+
+  phpunit-sqlite:
+    name: "PHPUnit with SQLite"
+    runs-on: ubuntu-18.04
+    # If true, allow this job to fail:
+    continue-on-error: true
+    strategy:
+      matrix:
+        # Define jobs for all combinations of values given to be tested:
+        php-version: ["5.6"]
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          coverage: "xdebug"
+
+      - name: Validate composer.json and composer.lock
+        run: composer validate
+
+      - name: Cache Composer packages
+        id: composer-cache
+        uses: actions/cache@v2
+        with:
+          path: vendor
+          key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-php-
+
+      - name: Install dependencies
+        run: composer install --no-progress
+
+      - name: Set up unit testing
+        run: .github/actions/setup.sh
+        env:
+          DB: sqlite
+
+      - name: Run unit tests
+        run: .github/actions/run_tests.sh
+        env:
+          DIR: ${{ github.workspace }}
+
+      - name: "Upload to Codecov"
+        uses: "codecov/codecov-action@v1"
+
+
+  phpunit-mariadb:
+    name: "PHPUnit with MariaDB"
+    runs-on: ubuntu-18.04
```

```suggestion
    runs-on: ubuntu-latest
```

Does this work? or do we need an older version of ubuntu for PHP5.X?

---

I actually had that originally and I think it worked fine. I thought it might be better to specify a version, as I assume "latest" will change over time, but if that's not a problem I'll (check and) change it back.

---

> I assume "latest" will change over time

I will, and I think we should only specify a version if, in the future, running on latest causes a problem.